tests/container/test/run-ci.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+export PATH=/home/test-user/receptor:/home/test-user/go/bin:$PATH
```

Comment:
Where is this script used?

Reply:
My mistake. I copied from GitHub Action workflow tests. That will be used when I finish the task called "Add command to run all tests on current source code".

Reply:
Done :)